src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -731,3 +732,9 @@ export function fetchFacilitySettings() {
     }
   };
 }
+
+export function updateBreadcrumb(breadcrumb) {
+  return async (dispatch, _getState) => {
```

Comment:
Just curious why use `_getState`? Other asynchronous calls have `getState` as the second param.

Reply:
'getState' is not being used so I could have deleted it. If you add an '_' before unused parameters eslint will ignore them. Sometimes I leave the unused parameters as a reminder.